rustls/examples/internal/bogo_shim_impl.rs

```diff
@@ -19,9 +19,9 @@ use rustls::{
 };
 
 #[cfg(all(not(feature = "ring"), feature = "aws_lc_rs"))]
-use rustls::crypto::{aws_lc_rs as provider, aws_lc_rs::AWS_LC_RS as PROVIDER};
+use rustls::crypto::aws_lc_rs as primary_provider;
```

Comment:
I think I'd just name this `provider`? `primary_provider` seems like a weird notion since I don't think there is another provider in play in the bogo shim at the same time? I guess you're trying to disambiguate from the `provider()` function but functions and modules live in a different namespace so `provider::provider()` should Just Work IIRC.

Reply:
> I guess you're trying to disambiguate from the provider() function

I think I picked this up from another module that was using `primary_provider` and thought it did a good job distinguishing from the provider() function, but as you mention maybe that's not necessary. 

I'll adjust to just use `primary` throughout :+1:

Reply:
Fixed.